### PR TITLE
FLT-1891 implement navigation shell

### PR DESCRIPTION
**Pull request by kawaic**
implement navigation per spec https://zpl.io/2yOPdpp

`npm run storybook`

go to `http://localhost:9009/?path=/story/dashboard--with-navigation`

### CODE REVIEW

File: src/components/home/Home.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+export const Header = styled.h1`
+  font-family: ${props => props.theme.fontFamiily};
```
Comment (abm-trov) [inline src/components/home/Home.styled.tsx]: Is the extra "i" in `fontFamiily` on purpose?

File: src/index.tsx [hunk 2]
```diff
@@ -2,18 +2,28 @@ import React from 'react';
...
 import * as serviceWorker from './serviceWorker';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import configureStore from './configureStore';
+import { GlobalStyle } from './index.styled';
 
 const { store, persistor } = configureStore();
+export const theme = {
+  primaryColor: '#0099da',
+  secondaryColor: '#9fa5a9',
+  fontFamiily: 'GalenoGrotesque',
```
Comment (abm-trov) [inline src/index.tsx]:
```suggestion
  fontFamily: 'GalanoGrotesque',
```

File: src/index.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import GalenoGrotesqueBold from './fonts/GalanoGrotesque/GalanoGrotesque-Bold.woff2';
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
import GalanoGrotesqueBold from './fonts/GalanoGrotesque/GalanoGrotesque-Bold.woff2';
```

File: src/index.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import GalenoGrotesqueSemiBold from './fonts/GalanoGrotesque/GalanoGrotesque-SemiBold.woff2';
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
import GalanoGrotesqueSemiBold from './fonts/GalanoGrotesque/GalanoGrotesque-SemiBold.woff2';
```

File: src/index.styled.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import GalanoGrotesque from './fonts/GalanoGrotesque/GalanoGrotesque-Regular.woff2';
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  @font-face {
+    font-family: GalenoGrotesque;
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
    font-family: GalanoGrotesque;
```

File: src/index.styled.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    src: url(${GalanoGrotesque}) format('woff2');
+  }
+  @font-face {
+    font-family: GalenoGrotesqueBold;
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
    font-family: GalanoGrotesqueBold;
```

File: src/index.styled.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    src: url(${GalenoGrotesqueBold}) format('woff2');
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
    src: url(${GalanoGrotesqueBold}) format('woff2');
```

File: src/index.styled.tsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  @font-face {
+    font-family: GalenoGrotesqueSemiBold;
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
    font-family: GalanoGrotesqueSemiBold;
```

File: src/index.styled.tsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    src: url(${GalenoGrotesqueSemiBold}) format('woff2');
```
Comment (abm-trov) [inline src/index.styled.tsx]:
```suggestion
    src: url(${GalanoGrotesqueSemiBold}) format('woff2');
```

File: src/components/home/Navigation.styled.tsx [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+`;
+
+export const TrovBrand = styled(NavbarBrand)`
+  font-family: ${props => props.theme.fontFamiily};
+  font-size: 14px !important;
+  color: ${props => props.theme.secondaryColor} !important;
+`;
+
+export const PartnerBrand = styled(NavbarBrand)`
+  font-family: 'GalenoGrotesqueSemiBold';
```
Comment (abm-trov) [inline src/components/home/Navigation.styled.tsx]:
```suggestion
  font-family: 'GalanoGrotesqueSemiBold';
```

File: src/components/home/Navigation.styled.tsx [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+  :hover {
+    color: ${props => props.theme.primaryColor} !important;
+  }
+`;
+
+export const SubmissionButton = styled(Button)`
+  color: #ffffff;
+  background-color: ${props => props.theme.primaryColor} !important;
+  text-transform: uppercase;
+  font-family: 'GalenoGrotesqueSemiBold';
```
Comment (abm-trov) [inline src/components/home/Navigation.styled.tsx]:
```suggestion
  font-family: 'GalanoGrotesqueSemiBold';
```

File: src/components/home/Navigation.styled.tsx [hunk 12]
```diff
@@ -0,0 +1,60 @@
+import { Navbar, DropdownToggle, NavbarBrand, NavItem, NavLink, Button } from 'reactstrap';
+import styled from 'styled-components';
+
+export const TopNav = styled(Navbar)`
+  font-family: ${props => props.theme.fontFamiily};
+  background-color: #f8f8f8;
+  padding-left: 105px !important;
```
Comment (abm-trov) [inline src/components/home/Navigation.styled.tsx]: It's a bit hard to tell, but does this make the top nav inset from the body, or is it full width?

Comment (kawaic) [reply src/components/home/Navigation.styled.tsx]: inset from the body, wasn't sure looking at zeplin how it should be.  
Screen Shot 2019-03-26 at 10 22 32 AM
should they be full width?  I can change it

File: src/components/home/Home.styled.tsx (hunk 1 above)
Comment (kawaic) [reply src/components/home/Home.styled.tsx]: no....

File: src/components/home/Navigation.styled.tsx (hunk 12 above)
Comment (abm-trov) [reply src/components/home/Navigation.styled.tsx]: The container should be at the break points they set in zeplin, but the top nav bar colors should be full width, i.e., the gray then white. I agree that it isn't super clear in zeplin; I'll ask Kevin to make another full width view.

Comment (kawaic) [reply src/components/home/Navigation.styled.tsx]: fixed

**Pull request merged**